src/app/login-data/service/user-login.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MyCookieBook/CookieBookFE/pullRequest?prid=7552947)
```suggestion
import { Injectable } from "@angular/core";
```